rust_sodium-sys/build.rs

```diff
@@ -257,7 +257,7 @@ fn main() {
                 let v = unwrap!(unwrap!(unwrap!(lines.next()).split('.').next()).parse::<u32>());
                 if v < 15 { "--disable-pie" } else { "" }
             } else {
-                ""
+                "--disable-pie"
             }
         }
         _ => "",
```

Comment:
@ffflorian, @ustulation: it is required to add `--disable-pie` on this line as well. I'm on Arch and I didn't have `lsb_release` binary at all, so the execution went down here and PIE was not disabled. As a result, I was unable to compile the rust_sodium. 

When I installed lsb_release, compilation began to succeed. While it is a sufficient workaround to have lsb_release installed, I'll create a PR to fix this.